native-maven-plugin/src/main/java/org/graalvm/buildtools/maven/NativeTestMojo.java

```diff
@@ -204,6 +207,16 @@ private void configureEnvironment() {
                         systemProperties.put(child.getName(), child.getValue());
                     }
                 }
+                Xpp3Dom runtimeArguments = dom.getChild("runtimeArgs");
```

Comment:
This would read the `<runtimeArgs>` configuration tag from the `maven-surefire-plugin` definition, which I don't think is what you want to do.

If what you want is Native Build Tool's `<runtimeArgs>`, then use the field annotated with `@Parameter`.